cluster/get-kube-local.sh

```diff
@@ -21,6 +21,7 @@ set -o nounset
 set -o pipefail
 
 KUBE_HOST=${KUBE_HOST:-localhost}
+KUBECONFIG_PATH={KUBECONFIG_PATH:-"/var/run/kubernetes"}
```

Comment:
this is confusing, makes it sound like this is the path to the kubeconfig file, not the containing dir